apps/orams/pages/EditProfilePage.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react'
+import { withRouter, Switch, Route, Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import { connect } from 'react-redux'
+import NotFound from 'shared/NotFound'
+import formProps from 'shared/formPropsSelector'
+import { loadProfile, updateProfile } from 'orams/actions/profileActions'
+import BusinessDetailsForm from 'orams/components/BusinessDetailsForm'
+import BusinessInfoForm from 'orams/components/BusinessInfoForm'
+import YourInfoForm from 'orams/components/YourInfoForm'
+import ToolsForm from 'orams/components/ToolsForm'
+import AwardsForm from 'orams/components/AwardsForm'
+import SubmitStepForm from 'orams/components/Submit'
+import FinishProfile from 'orams/components/FinishProfile'
+import LocalNav from 'shared/LocalNav'
+
+class EditProfilePage extends Component {
```

Comment:
we may need to refactor this page when we do profile view.  Maybe:
`/profile` - view profile
'/profile/edit` - edit business details
`/profile/edit/business-info`
but we can refactor when we do view profile